data/json/items/magazines.json

```diff
@@ -54,6 +54,38 @@
         ]
     },
     {   "type" : "MAGAZINE",
+        "id" : "lp_45_mag",
+        "price" : 5800,
+        "name" : "L39-45 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A compact light-weight magazine for use with L39-45 and L39B-45 pistols.",
+        "material": ["steel", "plastic"],
+        "volume" : 1,
+        "weight" : 70,
+        "to_hit" : -2,
+        "ammo_type" : "45",
+        "capacity" : 12,
+        "reliability" : 10,
+        "reload_time" : 50
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "lp_9mm_mag",
+        "price" : 5800,
+        "name" : "L39 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A compact light-weight magazine for use with L39 and L39B pistols.",
+        "material": ["steel", "plastic"],
+        "volume" : 1,
+        "weight" : 80,
+        "to_hit" : -2,
+        "ammo_type" : "9mm",
+        "capacity" : 21,
+        "reliability" : 10,
+        "reload_time" : 50
```

Comment:
probably best to start with all magazines at 100 for ease of later scaling (see other PRs)